docs/google/calibration.md

```diff
@@ -0,0 +1,186 @@
+
+# If you know the timestamp of a previous calibration, you can retrieve the
+# calibration using the timestamp
+calibration_name = ("projects/" + PROJECT_ID 
+                    + "/processors/" + PROCESSOR_ID
+                    + "/calibrations/" 
+                    + str(EPOCH_TIMESTAMP_SECONDS)) 
+previous_calibration = engine.get_calibration(calibration)
+
+# If you know the job-id, you can retrieve the calibration that the job used.
+job = engine.get_job("projects/" + PROJECT_ID
+                   + "/programs/"+PROGRAM_ID
+                   + "/jobs/" + JOB_ID)
+job_calibration = cg.EngineJob(cg.JobConfig(), job, engine).get_calibration()
+
+# The calibration can be iterated through using something like this:
+for metric_name in latest_calibration:
+  print(metric_name)
+  print('------')
+  for qubit_or_pair in latest_calibration[metric_name]:
+     # Note that although the value is often singular,
+     # The metric_value is of the type list and can have multiple values     
+     metric_value = latest_calibration[metric_name][qubit_or_pair]
+     print(f'{qubit_or_pair} = {metric_value}')
+```
+
+Calibration metrics will also soon be available from the 
+[Google Cloud Platform Console](https://console.cloud.google.com).
+
+## Total error and purity error
+
+Several metrics below define total error and purity error.  The total error
+measures the average error ε_a over all possible input states.  See Table 1 on
+page 11 of the [https://arxiv.org/abs/1910.11333](Supplementry Information)
+document for a description and comparison between average error, Pauli error,
+and depolarization error.
+
+The [Purity](https://en.wikipedia.org/wiki/Purity_(quantum_mechanics)) of the
+quantum state is often defined as the trace of the density matrix of the
+state after the operations (RB or XEB) have been applied.  The purity error,
+or impurity, is one minus the purity, scaled to be from zero to one.  This is
+often referred to as the normalized
+[Linear Entropy](https://en.wikipedia.org/wiki/Linear_entropy).
+after the randomized benchmark circuit has been applied.  
+
+The purity error can be interpreted as a measure of the incoherent error,
+such as those caused by stochastic processes such as relaxation.  The total
+error can be interpreted as containing both this incoherent error as well as
+the coherent, or control, error resulting from improper control or calibration
+of the device.
+
+Note that, due to statistical fluctuations, it is possible that the purity
+error can exceed the total error by small amounts.
+
+For more about purity benchmarking, see Section 6.3 of this
+[thesis](https://web.physics.ucsb.edu/~martinisgroup/theses/Chen2018.pdf) .
+
+## Individual Metrics
+
+Each metric can be referenced by its key in the calibration object, e.g.
+```latest_calibration['single_qubit_idle_t1_micros']```.
+
+Note that the metric names below are subject to change without notice.
+
+### P_00 readout error
```

Comment:
Why two 0s, when the metric key only has one?

Reply:
Fixed.  Note that both will appear for a while until I get everyone to switch over.

Reply:
But I think having just one 0 makes sense. I don't know what the second 0 means. P_00 could be interpreted as the probability of measuring a 0 given that you prepared a 0. But that's not what it means: it is the probability of measuring a 1!

Reply:
The idea is that the experiment is to prepare the state in 0, then measure it in 0, hence 00.  The error is a combination of both state preparation and measurement, so it's 1-p_00.  Though, now that I think about it, it does not include the rare case of the state being prepared incorrectly combined with an incorrect measurement.

People around here are coalescing around calling this p00 and p11 error, but I can raise it in the working group to see if there is a preference between p0 and p1 as well.

Reply:
I see, that makes sense.